lib/ios-crash-log.js

```diff
@@ -0,0 +1,61 @@
+import path from 'path';
+import _ from 'lodash';
+import _glob from 'glob';
+import { fs, getLogger } from 'appium-support';
+import B from 'bluebird';
+import { asyncmap } from 'asyncbox';
+
+const glob = B.promisify(_glob);
+
+
+require('date-utils');
```

Comment:
@imurchie it essentially monkeypatches `Date` when it's required; we should add a comment to explain why this isn't a normal require